### PR TITLE
Add linter and reformat js file

### PR DESCRIPTION
**Pull request by jazeee**
Add linting to help clean up initial bugs and format code. Fix an ambiguous `var` usage.

### CODE REVIEW

File: ui/API_scripts.js [hunk 1]
```diff
@@ -1,119 +1,120 @@
+// eslint-disable-next-line no-unused-vars
```
Comment (jazeee) [inline ui/API_scripts.js:1]: I'd view this while ignoring whitespace:
https://github.com/epic-power-rpg/roll20/pull/9/files?diff=unified&w=1

This is mostly auto-formatted via linter.

File: ui/API_scripts.js [hunk 2]
```diff
@@ -1,119 +1,120 @@
...
+    const only_positive = any_positive && !any_negative;
+    const only_negative = !any_positive && any_negative;
+    for (let i = 0; i < 3; i++) {
+      const die = randomInteger(6);
+      let roll;
+      if (die === 6) {
+        must_roll = true;
+        roll = '\u29C9'; // two joined squares character
+      } else {
+        roll = die - 3;
```
Comment (jazeee) [inline ui/API_scripts.js:19]: This inner `var` was in error. All `var`'s are hoisted to the root of the function, which makes this sort of override line 13, but in an ambiguous way.
The intention of the code here is to redefine `roll`, so simply removed `var` here.

File: ui/API_scripts.js [hunk 3]
```diff
@@ -1,119 +1,120 @@
...
+          roll = '<span style="color:red";>' + roll + '</span>';
+        } else if (roll >  0) {
+          any_positive = true;
+          roll = '<span style="color:blue";>+' + roll + '</span>';
+        }
+      }
+      if (text === '') {
+        text = String(roll);
+      } else {
+        text = `${text}, ${roll}`;
```
Comment (jazeee) [inline ui/API_scripts.js:37]: Converted to string template.

**Review by johnlamping — APPROVED** [review]
Thanks! 
I tried it in the Alanya campaign, and it still works.

**Pull request merged**